tests/README.md

```diff
@@ -179,9 +179,15 @@ internal_bench/bytebuf:
 
 SSL/TLS tests in `multi_net` and `net_inet` use a
 self-signed key/cert pair that is randomly generated and to be used for
-testing/demonstration only. You should always generate your own key/cert.
+testing/demonstration only.
 
-To generate a new self-signed RSA key/cert pair with openssl do:
+To use these on-device the der files should be copied and the current time
+set to ensure certs validity. This can be done with:
+```
+mpremote a0 rtc --set cp multi_net/*.der :
+```
+
+To instead generate a new self-signed RSA key/cert pair with openssl do:
```

Comment:
Suggest adding an extra bold-face text here for *You don't have to do these steps to run the tests successfully*. As it's otherwise natural if a new contributor sees a bunch of steps in a test README to think "oh, I should run these.".

Long term this feels like it belongs in a totally different place, maybe an example for making a TLS server with MicroPython? I don't think there's currently a suitable place, which I guess is how it ended up here in the first place.